Makefile

```diff
@@ -108,8 +108,8 @@ FETCH_SCRIPT        = $(BIN_DIR)/fetch_docs.js
 #      which includes them on its own, and the SCSS compiler takes care of them;
 #      because of this, there is also no .scss -> .css pattern rule
 ifdef WINDOWS
-SCSS_SRC   = $(subst $(PWD)/,,$(subst \,/,$(shell cd $(CSS_SRC_DIR) && dir *.scss /S /B)))
-STYLES_SRC = $(subst $(PWD)/,,$(subst \,/,$(shell cd $(CSS_SRC_DIR) && dir *.less *.css /S /B)))
+SCSS_SRC   = $(subst $(dir $(realpath $(firstword $(MAKEFILE_LIST)))),,$(subst \,/,$(shell cd $(CSS_SRC_DIR) && dir *.scss /S /B)))
+STYLES_SRC = $(subst $(dir $(realpath $(firstword $(MAKEFILE_LIST)))),,$(subst \,/,$(shell cd $(CSS_SRC_DIR) && dir *.less *.css /S /B)))
```

Comment:
Two comments:
1. It would be a bit more readable to put this computation into a variable (e.g. `PWD` or `CWD`), like so:

        CWD      = $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
        SCSS_SRC = $(subst $(CWD),,$( ...
2. Alternatively, you can use the `pwd` command, which should be available on Windows, like so:

        PWD = $(shell pwd)

Reply:
@dblotsky  Thanks for the review comments Dmitry. Have added PWD variable to make it more readable. $(shell pwd) does not work for me seamlessly. pwd works for me only in powershell. So the above solution is seamless i feel. Let me know your thoughts.